tikv-client/src/test/java/com/pingcap/tikv/expression/visitor/CanBePrunedValidatorTest.java

```diff
@@ -0,0 +1,74 @@
+package com.pingcap.tikv.expression.visitor;
+
+import static com.pingcap.tikv.expression.visitor.ExpressionTypeCoercerTest.createTable;
```

Comment:
Seems a bit strange to use createTable in another test... We are only using c1 and c3 here.